packages/scenario-harness/ui/app.js

```
@@ -434,6 +435,77 @@ function connectWebSocket() {
+    if (cb) cb.checked = (current & bit) !== 0;
+  }
+}
+
+function wireEngineWarnings() {
+  for (const { id, bit } of ENGINE_WARNINGS) {
+    const cb = $(id);
+    if (!cb) continue;
+    cb.addEventListener("change", () => {
+      const cur = state.controller?.telemetry?.EngineWarnings ?? 0;
+      const next = cb.checked ? cur | bit : cur & ~bit;
+      post("/api/telemetry", { patch: { EngineWarnings: next } }).catch((err) => alert(err.message));
```

ESLint

[error] 'alert' is not defined.

(no-undef)